go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mongodb/mongodb-atlas-cli/atlascli
 
-go 1.24.2
+go 1.24.3
```

Comment:
```suggestion
go 1.24.2
```

do you mind keeping it? our internal CI system does not support v1.24.3 yet.

Reply:
as soon as you change it, Ill rerun the tests

Reply:
also, we've recently updated unit tests on master, I'd apreciate if you could update your branch